Agent/Linux/ubuntu1804-README.md

```diff
@@ -0,0 +1,44 @@
+# Azure DevOps - Ubuntu 18.04 Agent
+
+The following sections provide a step-by-step guide to creating a Azure DevOps Agent Image using Packer, spin up an Agent using the Image and add it to the Azure DevOps Agent Pool.
+
+## Prepare your Azure DevOps Account
+
+1. Create an Azure DevOps Agent pool.
+
+2. Generate a Personal Access Token (PAT) for your Azure DevOps Organization. When generating the Personal Access Token (PAT), assign the following scopes:
+
+>a. Agent Pools - Read & Manage
+>b. Deployment Groups - Read & Manage
+
+3. Create a service principle and add it to your subscription as Owner. This is the subscription that will be used to host the image and Agent VM.
+
+4. Add the service principle to the subscription that will host the image and agent VMs with Owner role.
+
+## Build a Packer Image
+
+1. Build a Docker image and a container from the Docker image by mounting `Agent` folder as a volume in your Docker container.
+
+```
+docker run -it -v <path-to-vdc-folder>/Agent:/usr/app/src/Agent vdc:latest
+```
+
+2. Update the run.sh file with the service principle and subscription information.
+
+3. Run the command below from PowerShell terminal to build an VM Image
+
+```
+bash -c "./build.sh"
+```
+
+## Create a VM from Packer Image
+
+1. Run the command below from PowerShell terminal to create an instance from the VM Image
+
+```
+bash -c "./vsts-agent-create.sh <organization-url> <personal-access-token> <vsts-pool-name> <agent-vm-name>"
+```
+
+## Ubuntu Agent - Notes
+
+* PowerShell Modules - Az, Pester and Resource Graph are installed and available part of the Agent Image. These modules are installed using `Save-Module` Cmdlet. Ubuntu 18.04 has a specific location for these Modules to be saved. You will find the location of the Module to be saved in the `ubuntu-1804.json` packer file. Installing module using `Install-Module` will not allow the Cmdlets to be available for use by all users.
```

Comment:
Az Powershell Modules, switch the order of the words.

Also, add information on why you are using Save-Module instead of Install-Module list the issues that you'll face if using Install-Module

Reply:
Already have this as the last point of this paragraph. I separated this out into a separate line. There is really only one issue using Install-Module which is the inability of all the users to use the module.